Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1

```diff
@@ -288,6 +288,24 @@ Function Invoke-RebuildSolution {
     }
 }
 
+function Reset-EmptyTemplateDatabase { 
+  Invoke-Task -name ($MyInvocation.MyCommand.Name) -task {
+        $config = Get-DeployConfig
+        $ods = $config.databaseIds.ods
+        $connectionString = Get-DbConnectionStringBuilderFromTemplate -templateCSB $config.connectionStrings[$ods.connectionStringKey] -replacementTokens $config.emptyTemplateSuffix
```

Comment:
Let's just hardcode `Ods_Sandbox_Empty` instead of $config.emptyTemplateSuffix and then rename the function Reset-EmptySandboxDatabase. Since this is really just a helper function for setting up an empty I think this is okay. This will make it a bit easier to wire up the api client in the admin database and allow you to connect to this database correctly while running in sandbox mode.
We should probably go ahead and clean up references to $config.emptyTemplateSuffix since we won't be using it anymore.

```suggestion
        $connectionString = Get-DbConnectionStringBuilderFromTemplate -templateCSB $config.connectionStrings[$ods.connectionStringKey] -replacementTokens 'Ods_Sandbox_Empty'
```

Reply:
I removed emptyTemplateSuffix variable in all reference and updated to 'Ods_Sandbox_Empty'